src/streams/StreamScreen.js

```diff
@@ -76,8 +76,8 @@ class StreamScreen extends PureComponent<Props> {
           onValueChange={this.toggleStreamPushNotification}
           customStyle={this.context.styles.backgroundColor}
         />
-        <ZulipButton text="Topics" onPress={this.handleTopics} />
-        <ZulipButton text="Edit" onPress={this.handleEdit} />
+        <ZulipButton style={this.context.styles.marginTop} text="Topics" onPress={this.handleTopics} />
+        <ZulipButton style={this.context.styles.marginTop} text="Edit" onPress={this.handleEdit} />
```

Comment:
prefer-destructuring, like
`const { styles } = this.context;` 
and then
`styles.marginTop`